state/test/trie/test_prefix_nodes.py

```diff
@@ -20,7 +20,7 @@ def test_get_prefix_nodes():
     # The last node should be a leaf since only 1 key
     assert trie._get_node_type(last_node) == NODE_TYPE_LEAF
     # Seen prefix matches the prefix exactly
-    assert seen_prefix == prefix_nibbles
+    assert seen_prefix == []
```

Comment:
So, do we have a test where we actually fill seen_prefix?

Reply:
Ok